Rectangle/uk.lproj/Main.strings

```diff
@@ -1,550 +1,550 @@
+"xcE-uL-2J0.title" = "Вибір Spectacle також циклічно переключає 1/2, 2/3 і 1/3 ширини вікна при повторному використанні комбінації клавіш";
 
 /* Class = "NSMenuItem"; title = "Quit Rectangle"; ObjectID = "yvN-PE-bxn"; */
-"yvN-PE-bxn.title" = "Quit Rectangle";
-
-"Halves" = "Halves";
-"Corners" = "Corners";
-"Thirds" = "Thirds";
-"Maximize" = "Maximize";
-"Size" = "Size";
-"Display" = "Display";
-"Other" = "Other";
-"Move to Edge" = "Move to Edge";
-"Fourths" = "Fourths";
-"Sixths" = "Sixths";
-
-"Default Shortcuts" = "Default Shortcuts";
-"Currently using: " = "Currently using: ";
-"Cancel" = "Cancel";
-
-"Use frontmost.app as Todo App" = "Use frontmost.app as Todo App";
-"Enable Todo Mode" = "Enable Todo Mode";
-"Reflow Todo" = "Reflow Todo";
-
-"Left half, top/bottom half near corners" = "Left half, top/bottom half near corners";
-"Right half, top/bottom half near corners" = "Right half, top/bottom half near corners";
-"Thirds, drag toward center for two thirds" = "Thirds, drag toward center for two thirds";
-"Thirds, top/bottom half near corners" = "Thirds, top/bottom half near corners";
-"Left or right half" = "Left or right half";
-"Top sixths from corners; maximize" = "Top sixths from corners; maximize";
-"Bottom sixths from corners; thirds" = "Bottom sixths from corners; thirds";
-"Fourths columns" = "Fourths columns";
-
-"Go to System Settings → Privacy & Security → Accessibility" = "Go to System Settings → Privacy & Security → Accessibility";
-
-"Open System Settings" = "Open System Settings";
-
-"Use as Todo Window" = "Use as Todo Window";
+"yvN-PE-bxn.title" = "Вийти з Rectangle";
+
+"Halves" = "Половини";
+"Corners" = "Кути";
+"Thirds" = "Третини";
+"Maximize" = "Максимізувати";
+"Size" = "Розмір";
+"Display" = "Дисплей";
+"Other" = "Інше";
+"Move to Edge" = "Перемістити до краю";
+"Fourths" = "Четверини";
+"Sixths" = "Шості";
+
+"Default Shortcuts" = "Типові комбінації клавіш";
+"Currently using: " = "Використовується: ";
+"Cancel" = "Скасувати";
+
+"Use frontmost.app as Todo App" = "Використовувати frontmost.app як застосунок Завдань";
+"Enable Todo Mode" = "Увімкнути режим Завдань";
+"Reflow Todo" = "Переформатувати вікна Завдань";
+
+"Left half, top/bottom half near corners" = "Ліва половина, верхня/нижня половина близько кутів";
+"Right half, top/bottom half near corners" = "Права половина, верхня/нижня половина близько кутів";
+"Thirds, drag toward center for two thirds" = "Третини, перетягніть у центр для двох третин";
+"Thirds, top/bottom half near corners" = "Третини, верхня/нижня половина близько кутів";
+"Left or right half" = "Ліва або права половина";
+"Top sixths from corners; maximize" = "Верхні шості від кутів; максимізувати";
+"Bottom sixths from corners; thirds" = "Нижні шості від кутів; третини";
+"Fourths columns" = "Четвертини стовпців";
+
+"Go to System Settings → Privacy & Security → Accessibility" =
```

Comment:
Everything else looks good, just this have this missing one.

Reply:
oh, sorry, missed that somehow. Fixed now